internal/controllers/utils/utils.go

```diff
@@ -1044,21 +1042,7 @@ func ClusterIsReadyForPolicyConfig(
 		available = true
 	}
 
-	acceptedCondition := meta.FindStatusCondition(
-		managedCluster.Status.Conditions,
-		clusterv1.ManagedClusterConditionHubAccepted)
-	if acceptedCondition != nil && acceptedCondition.Status == metav1.ConditionTrue {
-		hubAccepted = true
-	}
-
-	joinedCondition := meta.FindStatusCondition(
-		managedCluster.Status.Conditions,
-		clusterv1.ManagedClusterConditionJoined)
-	if joinedCondition != nil && joinedCondition.Status == metav1.ConditionTrue {
-		joined = true
-	}
-
-	return available && hubAccepted && joined, nil
+	return available, nil
```

Comment:
Why shouldn't we also check for joined and accepted? :thinking:

Reply:
I changed it to pass 2 unit test cases, since they only set the available condition:
- "Verify status conditions when ClusterInstance provision has completed, ManagedCluster becomes ready and configuration policy becomes compliant"
- "Verify status conditions when configuration change causes ClusterInstance rendering to fail but configuration policy becomes compliant"

see: https://github.com/openshift-kni/oran-o2ims/blob/a617e9c002be312d3c353b6db861106acf07c6b5/internal/controllers/clusterrequest_controller_test.go#L1026

Reply:
Fixed. I re-added the joined and accepted condition check and modified the existing unit test